docs_user/modules/proc_adopting-image-service-with-nfs-ganesha-backend.adoc

```diff
@@ -25,7 +23,7 @@ In particular, find among the {OpenStackPreviousInstaller} heat templates the fo
 
 ----
 +
-In the example above, as the first variable shows, the {image_service} has no notion of NFS Ganesha backend: the `File` driver is used in this scenario, and behind the scenes, the `filesystem_store_datadir` which usually points to `/var/lib/glance/images/` is mapped to the export value provided by the `GlanceNfsShare` variable.
+In the example above, as the first variable shows, the {image_service} has no notion of NFS Ganesha back end: the `File` driver is used in this scenario, and behind the scenes, the `filesystem_store_datadir` which usually points to `/var/lib/glance/images/` is mapped to the export value provided by the `GlanceNfsShare` variable.
```

Comment:
The whole section that starts with “In the example above…” seems like background information. In the prerequisites section, we usually want to keep the information focused on the action that the customer needs to take. Is this information necessary for downstream? Can we put it in its own module in this assembly?

Reply:
wait, why this mentions "Ganesha" ? the NFS share is not provided by `Ganesha` (and there's no Ceph involved here), which is only used by `Manila` only. This is a "generic" NFS service, so I would rename it and avoid mentioning Ganesha.
This is a background information that clarifies the fact that in general Glance has no notion of NFS: it uses a `File` (aka `local`) backend and the underlying filesystem points to the /var/lib/glance/images that is mapped to this generic, external, nfs share.
I think we can turn it into a `Note` if it helps, but considering that customers are doing a brownfield operation (moving from the baremetal world to openshift), it's important they understand what they are doing (without too much detail of course).
Do you think we can rework this sentence into a Note instead of another assembly (other than fixing the "Ganesha" part)?

Reply:
OK, I changed this to a note and edited the text. I also removed references to "Ganesha" throughout the file.